charts/finrem-ns/values.yaml

```diff
@@ -1,15 +1,9 @@
 java:
+  ingressHost: "finrem-ns-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
   applicationPort: '8086'
   livenessPath: '/health'
-  environment:
-      AUTH_PROVIDER_SERVICE_CLIENT_MICROSERVICE: 'finrem-notification-service'
-      REFORM_SERVICE_NAME: 'ns'
-      AUTH_PROVIDER_SERVICE_CLIENT_TOKENTIMETOLIVEINSECONDS: '900'
-      REFORM_TEAM: 'finrem'
-  image: 'https://hmcts.azurecr.io/hmcts/finrem-notification-service:latest'
   keyVaults:
     finrem:
       resourceGroup: finrem
       secrets:
         - gov-uk-notification-key
-  aadIdentityName: finrem
```

Comment:
this is needed it's setting the default for all envs

Reply:
ok, since it's also in yaml file in cnp-flux-config - are both needed - `aadIdentityName` in flux config and in `values.yaml` ?

Reply:
once you've released this chart it's not needed in cnp-flux-config

Reply:
restored